lib/mix/tasks/docs.ex

```diff
@@ -164,6 +165,7 @@ defmodule Mix.Tasks.Docs do
     Keyword.put_new(options, :source_beam, compile_path)
   end
 
+  @lint {Credo.Check.Design.TagTODO, false}
```

Comment:
Same here. TODO is disabled in the credo.exs (there are other things that checks TODOs in ebert).